src/ray/rpc/gcs_server/gcs_rpc_client.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <thread>
+
+#include <grpcpp/grpcpp.h>
+
+#include "src/ray/protobuf/gcs_service.pb.h"
+#include "src/ray/rpc/client_call.h"
+
+namespace ray {
+namespace rpc {
+
+/// Client used for communicating with a remote job info access server.
```

Comment:
This will eventually implement some Jobs interface right?

Reply:
> This will eventually implement some Jobs interface right?

Sorry, my bad! The comment is a bit of confuse. All interface(Jobs/Nodes/Actors ...) will be implemented inside GcsRpcClient in the future.